backend/server/src/models/requestModel.ts

```diff
@@ -0,0 +1,84 @@
+import type { MatchType, PlayerColor, PointType } from "./matchModel.js";
+
+/**
+ * MongoDB Object ID.
+ * Represents a 24-character hexadecimal string.
+ * Used for the purpose of identifying and handling invalid ID query parameters prior to initiating queries.
+ * @pattern ^[0-9a-fA-F]{24}$
+ * @format objectId
+ */
+export type ObjectIdString = string;
```

Comment:
This is just an alias for string type but can be used to validate query parameters.